events/ME_Rum_Events.txt

```diff
@@ -30,19 +30,19 @@ country_event = {
 				}
 			}
 		}
-		if = {
-			limit = {
-				is_expanded_mod_active = { mod = subjects }
-				can_be_overlord = muslim_tributary_vassal
-				TRA = {
-					NOT = { religion_group = muslim }
-				}
-			}
-			create_subject = {
-				subject = TRA
-				subject_type = muslim_tributary_vassal
-			}
-		}
+		#if = {
+		#	limit = {
+		#		is_expanded_mod_active = { mod = subjects }
+		#		can_be_overlord = muslim_tributary_vassal
+		#		TRA = {
+		#			NOT = { religion_group = muslim }
+		#		}
+		#	}
+		#	create_subject = {
+		#		subject = TRA
+		#		subject_type = muslim_tributary_vassal
+		#	}
+		#}
 		else_if = {
```

Comment:
should be if

Reply:
There is an `if` above the commented section.

Reply:
Right but this else_if shouldn't be nested into that one.

Reply:
Right but this else_if shouldn't be nested into that one.

Reply:
Changed.